examples/common/scripts/vttablet-down.sh

```diff
@@ -19,12 +19,7 @@
 
 source "$(dirname "${BASH_SOURCE[0]:-$0}")/../env.sh"
 
-printf -v tablet_dir 'vt_%010d' $TABLET_UID
-pid=`cat $VTDATAROOT/$tablet_dir/vttablet.pid`
-
-kill $pid
-
-# Wait for vttablet to die.
-while ps -p $pid > /dev/null; do sleep 1; done
+printf -v tablet_dir 'vt_%010d' "$TABLET_UID"
 
+stop_process "vttablet" "$VTDATAROOT/$tablet_dir/vttablet.pid"
```

Comment:
Not completely a nit: it might be clearer if we replace `"vttablet"` by `"vttablet $TABLET_UID"`.

Reply:
Good point. Will add that. Thanks!

Reply:
Actually, we already have that in the output so I will leave it as is:
```
Shutting down tablet zone1-0000000100
Stopping vttablet...
```